### DISCUSSION TITLE
Disable a Job

### DISCUSSION THREAD

**Discussion by neilatuniofyorkspace** [quartznet/quartznet#1836] [Q&A] [answered]
Is it possible to temporary /  permanently disable a job within Quartz.NET ?

Possibly disable / remove it from the Trigger* tables ?  

Many Thanks.

**Comment by lahma** [comment, marked as answer]
Triggers and jobs are a different thing. If you want to disable trigger/job temporarily there are `IScheduler.PauseTrigger`/`IScheduler.PauseJob` and `IScheduler.ResumeTrigger`/`IScheduler.ResumeJob`.
